### ISSUE TITLE
Forward listener events to PBS

### ISSUE THREAD

**Issue by clho40** [prebid/prebid-mobile-android#474]
Hello all,

I have a simple banner view implemented, as follows:
```
private fun createBanner() {
        val bannerView = BannerView(applicationContext, _configId, AdSize(320,50))
        bannerView.setBannerListener(object : BannerViewListener {
            override fun onAdLoaded(p0: BannerView?) {
                Toast.makeText(applicationContext, "onAdLoaded", Toast.LENGTH_SHORT).show()
            }

            override fun onAdDisplayed(p0: BannerView?) {
                Toast.makeText(applicationContext, "onAdDisplayed", Toast.LENGTH_SHORT).show()
            }

            override fun onAdFailed(p0: BannerView?, p1: AdException?) {
                Toast.makeText(applicationContext, "onAdFailed - ${p1.toString()}", Toast.LENGTH_SHORT).show()
            }

            override fun onAdClicked(p0: BannerView?) {
                Toast.makeText(applicationContext, "onAdClicked", Toast.LENGTH_SHORT).show()
            }

            override fun onAdClosed(p0: BannerView?) {
                Toast.makeText(applicationContext, "onAdClosed", Toast.LENGTH_SHORT).show()
            }
        })

        binding.adFrame.addView(bannerView)
        bannerView.loadAd()
    }
```
Can we, somehow, "listen" to these events on PBS too? If I can capture the click event on the PBS side I can forward the click/ impression events to our adx server?

**Comment by YuriyVelichkoPI** [comment]
Hi @clho40! 

Here you can find which events are supported by PBS:
https://docs.prebid.org/prebid-server/endpoints/pbs-endpoint-event.html

Starting with version 2.0.1 Prebid SDK will support sending `win` and `imp` events at the respective moments of the ad lifecycle. So you can listen to these events on PBS. I'd suggest asking the PBS team about how to do it.

There is no support for the `click` event on PBS so far. 
Video events might be supported later in the SDK.

**Comment by clho40** [comment]
@YuriyVelichkoPI Thank you for the good news :) Any idea when version 2.0.1 will be released? Are there any materials I could start reading now?

**Comment by YuriyVelichkoPI** [comment]
We are going to release 2.0.1 this or next week. 

I haven't found the exact guide for events. In order to enable events we added the following lines to the `app-settings.yaml`: 

```
accounts:
  - id: 1001
    auction:
      events:
        enabled: true
```

Make sure that `id` is an account id in your bid request. 

Also, you should set the URL of your PBS  into `${EXTERNAL_URL}` property in the `config.yaml`. After that, you should see the respective events in the bid response. SDK will handle them.

**Comment by YuriyVelichkoPI** [comment]
The support of `win` and `imp` events was introduced in the scope of this task 

https://github.com/prebid/prebid-mobile-android/issues/443

**Comment by clho40** [comment]
Hello @YuriyVelichkoPI , sorry I did not follow up on this earlier. I am still confused as to how should we implement this on our side.

1. On the Prebid Server,
- The adapter includes the win url on `seatbid.bid.nurl`, but from this issue https://github.com/prebid/prebid-server/issues/1202 I see that we have to use `wurl` and I don't see `wurl` as part of the ORTB protocol. Am I missing something?

2. On the Prebid Mobile SDK,
From your post - https://github.com/prebid/prebid-mobile-android/issues/443 I see that the `win` and `imp` reporting feature has been implemented. But how do we make the win url (not sure from `nurl` or `wurl`) appear under `response.seatbid.bid.ext.prebid.events.win` and `response.seatbid.bid.ext.prebid.events.imp`?

And, assuming that we are able to make 1. and 2. work, would the Mobile SDK invoke the win url or the PBS?
